packages/yum/centos-8/Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG FROM=centos:8
```

Comment:
Could you remove this file?

Reply:
Ah...Yes, this is needless...
It has arleady been supported at https://github.com/groonga/groonga/pull/1298/commits/50ed56c6630242ff86b9513041afd570b3e925df.
Thank you for your support.